File: java8/1074.java
import java.util.Scanner;
public class Main {

	public static void main(String[] args) {
		Scanner readOnTheKeyboard = new Scanner (System.in);

		String q = readOnTheKeyboard.nextLine();
		
		int w = Integer.parseInt(q);
		
		while(w > 0) {
		
		String a = readOnTheKeyboard.nextLine();
		int A = Integer.parseInt(a);
		
		if (A == 0) {
			System.out.println("NULL");}
		
		else if (A < 0) {
			if (A % 2 == 0) {
				System.out.println("EVEN NEGATIVE");}
			else {
				System.out.println("ODD NEGATIVE");}
		}
		
		else if (A > 0) {
			if (A % 2 == 0) {
				System.out.println("EVEN POSITIVE");}
			else {
				System.out.println("ODD POSITIVE");}
		}
		
		w--;
		}
		
		readOnTheKeyboard.close();
//Precisa de lista
	}
}

File: python3/1074.py
x = int (input())
l=[]
for k in range(x):
    q=int(input())
    l.append(q)


for k in l:
    
    if (k == 0):
        print("NULL")
		
    elif(k < 0):
        if (k % 2 == 0):
            print("EVEN NEGATIVE")
        else:
            print("ODD NEGATIVE")
		
    elif (k > 0):
        if (k % 2 == 0):
            print("EVEN POSITIVE")
        else:
            print("ODD POSITIVE")